src/sass/layout/_contacts.scss

```diff
@@ -0,0 +1,298 @@
+    padding: 4px 13px;
+    font-size: 16px;
+    line-height: calc(22 / 16);
+    letter-spacing: 0.04em;
+    margin-bottom: 32px;
+    max-height: 30px;
+  }
+
+  @include desktop {
+    padding: 4px 14px;
+    margin-bottom: 32px;
+    max-height: 30px;
+  }
+}
+
+.contacts__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(22 / 12);
+  letter-spacing: 0.04em;
+  color: getColor('theme-black');
+  margin-bottom: 8px;
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
+  }
+
+  @include desktop {
+    font-size: 16px;
+    line-height: calc(30 / 16);
+  }
+}
+
+.contacts__addresses {
+  margin-bottom: 24px;
+
+  @include tablet;
+  @include desktop {
+    margin-bottom: 32px;
+  }
+}
+
+.contacts__link-orange {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(22 / 12);
+  letter-spacing: 0.04em;
+  color: getColor('orange-main');
+  cursor: pointer;
+
+  &:hover,
+  &:focus {
+    text-decoration-line: underline;
+  }
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
+  }
+
+  @include desktop {
+    font-size: 16px;
+    line-height: calc(29 / 16);
+  }
+}
+
+.work-shedule {
+  border-top: 1px solid #e1e1e1;
+  border-bottom: 1px solid #e1e1e1;
+  padding: 24px 0px;
+  margin-bottom: 24px;
+
+  @include tablet;
+  @include desktop {
+    padding: 32px 0px;
+    margin-bottom: 32px;
+  }
+}
+
+.work-shedule__item:not(:last-child) {
+  margin-bottom: 8px;
+}
+
+.work-shedule__text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(22 / 12);
+  color: getColor('theme-black');
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
+  }
+
+  @include desktop {
+    font-size: 16px;
+    line-height: calc(30 / 16);
+  }
+}
+
+.work-shedule__time {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(22 / 12);
+  color: getColor('theme-grey-2');
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
+  }
+
+  @include desktop {
+    font-size: 16px;
+    line-height: calc(30 / 16);
+  }
+  & + span {
+    font-weight: 500;
+    color: getColor('theme-grey-2');
+  }
+}
+
+.contacts__connection {
+  max-width: 130px;
+  max-height: 44px;
+
+  @include desktop {
+    max-width: 173px;
+    max-height: 52px;
+  }
+}
+
+.contacts__link {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(19 / 12);
+  letter-spacing: normal;
+  color: getColor('theme-black');
+  cursor: pointer;
+
+  &:hover,
+  &:focus {
+    color: getColor('theme-black');
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 6px;
+  }
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
+  }
+
+  @include desktop {
+    font-size: 16px;
+    line-height: calc(24 / 16);
+  }
+}
+
+.button__group {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  @include tablet {
+    flex-direction: row;
+    justify-content: center;
+    align-items: stretch;
+    margin-right: 16px;
```

Comment:
margin is for the whole group, not for one button. should be written for button__first.

Reply:
fixed